src/components/CloudStatusOverviewMultiRegionPage.js

```diff
@@ -9,33 +9,18 @@ import CloudStatusSidebar from './CloudStatusSidebar';
 import StatusDataPeriodPicker from './StatusDataPeriodPicker';
 import Score from './Score';
 
-@observer(['uiState', 'regions', 'regionAvailabilityData', 'regionHealthData'])
+@observer(['uiState', 'regions', 'regionOverviewData'])
 export default class CloudStatusOverviewMultiRegionPage extends Component {
   static async fetchData(
-    {uiState, regionAvailabilityData, regionHealthData},
+    {uiState, regionOverviewData},
     {dataPeriod = uiState.activeStatusDataPeriod} = {}
     ) {
     const url = `/api/v1/status/${encodeURIComponent(dataPeriod)}`;
     const response = await fetch(url);
     const responseBody = await response.json();
     transaction(() => {
       forEach(responseBody.status, (plainRegionStatusData, regionName) => {
```

Comment:
I would rename then plainRegionStatusData to plainRegionOverviewData, if plainRegionStatusData now contains data for a region overview only.